app/assets/javascripts/autocomplete.js.coffee

```diff
@@ -50,7 +50,7 @@
       target.val ""
     else if $t.data('matched_val') != typed
       mySource.remote.get typed, (matches) ->
-        if matches.length > 0
+        if matches.length == 1
           target.val matches[0].id
         else if !$validate
```

Comment:
https://github.com/ualbertalib/avalon/blob/avalon-6/app/views/media_objects/_resource_description.html.erb#L48 will toggle this value.  This might be useful if the preference is to leave this value blank rather that using the user entered text.